### PR TITLE
Support Statamic 4

### PR DESCRIPTION
**Pull request by sunscreem**
This isn't tested, I just created my own fork, pointed my composer to that fork and `composer update` - everything appears to be working correctly.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   "require": {
     "php": "^7.3 || ^8.0",
     "illuminate/support": "^8.0 || ^9.0",
-    "statamic/cms": "3.0.* || 3.1.* || 3.2.* || 3.3.* || 3.4.*"
+    "statamic/cms": "3.0.* || 3.1.* || 3.2.* || 3.3.* || 3.4.* || 4.* "
```
Comment (edalzell) [inline composer.json]: This can be `"^3.0 || ^4.0" now.

File: composer.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
   "require": {
     "php": "^7.3 || ^8.0",
     "illuminate/support": "^8.0 || ^9.0",
```
Comment (edalzell) [inline composer.json]: This needs to be `"^8.0 || ^9.0" || ^10.0"`

**Comment by jonassiewertsen** [comment]
Thanks @sunscreem

**Comment by jonassiewertsen** [comment]
I did not see this PR before. 

Those changes have been published with version 1.8.0. 

Let me know if things run smoothly, as I did not test those changes.

**Pull request merged**